packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -0,0 +1,29 @@
+import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
+import BigNumber from 'bignumber.js'
+
+// Disabling no-return-assign makes the code cleaner with the setter and getter */
+/* eslint-disable no-return-assign */
+
+/**
+ * DestroyLoanScheme DeFi Transaction
+ */
+export interface DestroyLoanScheme {
+  identifier: string // ------------------| c = VarUInt{1-9 bytes}, + c bytes UTF encoded string
+  height: BigNumber // -------------------| 8 bytes unsigned integer
```

Comment:
`height` is optional in the RPC. with default value 0.
refer https://github.com/DeFiCh/ain/blob/8e9de26cfc32cb5a7183a95fda9b0c016f275486/src/masternodes/rpc_loan.cpp#L803
https://github.com/DeFiCh/ain/blob/8e9de26cfc32cb5a7183a95fda9b0c016f275486/src/masternodes/loan.h#L172

so maybe change this optional. and make sure `DestroyLoanScheme.height = DestroyLoanScheme.height ?? 0x00000000` at transaction builder layer. for discussion.